src/features/multipleChoiceAnswer/MultipleChoiceAnswer.tsx

```diff
@@ -0,0 +1,86 @@
+import { css } from '@emotion/react';
+
+import colors from '~/styles/color';
+import { BODY_2_BOLD } from '~/styles/typo';
+
+const MultipleChoiceAnswer = ({ variant = 'highlighted', totalCount = 3, answeredCount = 1 }) => {
```

Comment:
현재 상태에서는 variant가 string으로 타입 추론되지 않나요??

```suggestion
interface Props {
  variants?: 'highlighted' | 'default';
  totalCount: number
  answeredCount: number;
}

const MultipleChoiceAnswer = ({ variant = 'highlighted', totalCount, answeredCount }: Props) => {
```

이렇게 타입을 선언하면 어떨까요 ?!

Reply:
오 네네 맞아요! 뭔가 찝찝했었는데 놓쳤네요!! 집어주셔서 감사합니다 ㅎㅎ